### ISSUE TITLE
Can you release the code for training?

### ISSUE THREAD

**Issue by hszyq1234** [fabvio/Cascade-LD#1]

**Comment by mengzhibin** [comment]
Thanks for you great work, look forward for your training hyper parameters releasing.

**Comment by fabvio** [comment]
Sorry for the delay, I am currently out of office for holidays so it's a little bit difficult to release the training code.

About the code, it's messy and should be reorganized in order to be fully usable, but I plan to do that in the near future. Meanwhile, I will first release my implementation of Hsu clustering loss. With that it should be easy to reproduce at least the first part of the paper. Stay tuned.

**Comment by anand-kummari** [comment]
what is the loss function you have used to train the model. Can you release the loss function or give insights on how to implement it?

**Comment by fabvio** [comment]
I released the loss function in the latest commit.

**Comment by noobliang** [comment]
Hi， can you release the training code  ,thanks

**Comment by nieyan** [comment]
Hi,  which version of pytorch do you use ?

**Comment by fabvio** [comment]
I'm ordering the code but only in my free time so release will take a little longer, also I will probably write a journal extended version of the paper so I will explain better the procedures having broader page limitations.

About the pytorch version, the code has been deployed on pytorch <= 0.4.0, but it should work also on >= 0.4.0.

**Comment by kmsgnnew** [comment]
@fabvio its great !!! will the code be available soon?

**Comment by nikhilbv** [comment]
+1

**Comment by rashmi0888** [comment]
Hi @fabvio . Can you release pretrained weights for descriptor size = 256 or 128?

**Comment by nikhilbv** [comment]
Hi @fabvio, I Would love to know when the training code will be released?

**Comment by sandeepnmenon** [comment]
+1
Would love to know when the training code will be released?

**Comment by ds12a** [comment]
@fabvio Any updates on when the training code will be released?

**Comment by fabvio** [comment]
Unfortunately, we stopped the development due to more urgent work. Anyways, I'd be happy to guide whoever wants to reimplement the method from scratch. Also, given that it could be adapted to any instance-based lane detector, it could also be plugged in to more recent baselines. You can contact me in private.
